src/modules/DogsByBreed.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { useSelector,useDispatch } from 'react-redux'
+import { addDogToMyTeam } from '../reducers/myTeamReducer'
+
+
+
+const DogsByBreed=({dogs})=>{
+
+  const dispatch=useDispatch()
+  const breedToShow=useSelector(state=>state.breedToShow) 
+  const myTeam=useSelector(state=>state.myTeam)
+  
+    if (dogs.length>0)
+    return (
+      <div>
+        <h3>{breedToShow}</h3>
+        {dogs.map(d =>
+          <div key={d}>
+        <img src={d} alt="dog img"width='200'height='200' />
+        <button onClick={()=>{
+         const dogWithSameBreed=myTeam.filter(perro=>perro.breed===breedToShow)
+            
+         if (dogWithSameBreed.length===3) {
```

Comment:
Poner esta logica aca es discutible. No esta mal, pero puede hacerse de otra manera. Si exisitiera otroa componente que tambien puede agregar perros al equipo, entonces te verias obligado a repetir toda esta logica dentro del mismo (en tu caso no pasa)
Una alternativa es meter esta logica dentro del reducer, para que no tengas que andar repitiendo la logica dentro de cada componente que hagas que quiera agregar perros al team.
Aca podes ver una discusion al respecto: https://stackoverflow.com/questions/43848257/where-to-put-business-logic-in-redux-action-or-store